### PR TITLE
[12x.] Add month enum to scheduler cron method yearlyOn()

### PR DESCRIPTION
**Pull request by shaedrich**
Continuation of #53961

### CODE REVIEW

File: src/Illuminate/Console/Enums/ScheduledMonth.php [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Illuminate\Console\Enums;
+
+enum ScheduledMonth: string
```
Comment (crynobone) [inline src/Illuminate/Console/Enums/ScheduledMonth.php:5]:
```suggestion
enum ScheduledMonth: int
```

Comment (shaedrich) [reply src/Illuminate/Console/Enums/ScheduledMonth.php:5]: Damn, good catch 👍🏻 Thought, I changed it 🤦🏻

**Comment by taylorotwell** [comment]
This stuff doesn't really need to go to master.

**Comment by shaedrich** [comment]
> This stuff doesn't really need to go to master.

@taylorotwell So, `11.x` then? oO

**Comment by shaedrich** [comment]
see #53983

**Pull request closed without merging**